docs/Heroku.md

```diff
@@ -4,7 +4,7 @@ cd stringer
 heroku create
 git push heroku master
 
-heroku config:set APP_URL=`heroku apps:info | grep -o 'http[^"]*'`
+heroku config:set APP_URL=`heroku apps:info --shell | grep web_url | cut -d= -f2`
```

Comment:
I'm not sure when the `--shell` option was introduced. However, seeing as recent versions of the Heroku toolbelt include automatic updates, I don't think backwards comparability will be a problem.